about.html

```diff
@@ -0,0 +1,129 @@
+    <link rel="stylesheet" href="./about_style.css" />
+    <!--Icons-->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <div id="home-background">
+      <header>
+        <div class="hamburger-menu">
+          <input id="menu__toggle" type="checkbox" placeholder="input" />
+          <label class="menu__btn" for="menu__toggle">
+            <span class="span"></span>
+          </label>
+
+          <ul class="menu__box">
+            <li><a class="menu__item" href="./index.html">Home</a></li>
+            <li><a class="menu__item" href="./about.html">About</a></li>
+          </ul>
+        </div>
+        <nav class="top-var1">
+          <a href="https://github.com/nimplay">
+            <img class="icon-top-bar" src="./images/Github.png" alt="github"/></a>
+          <a href="https://twitter.com/NimrodAcosta"><img
+              class="icon-top-bar"
+              src="./images/Icon-Twitter.png"
+              alt="twitter"
+          /></a>
+          <a class="top-bar1-p" href="">Login/Register</a>
+        </nav>
+        <nav class="top-var2">
+          <div id="logo-icon">
+            <a href="./index.html"
+              ><img id="main-logo" src="./images/nimplayLogo.jpg" alt="Nimplay"
+            /></a>
+          </div>
+          <div id="main-bar2">
+            <a href="./index.html" class="text-orange top-text-var">Home</a>
+            <a href="./about.html" class="text-gray top-text-var">About</a>
+            <a id="button-main-bar" href="https://nimplay.itch.io/">Jam</a>
+          </div>
+        </nav>
+      </header>
+      <section class="home" id="hero">
+        <div class="home-titles">
+          <h2 id="hello" class="text-orange">"Hello world"</h2>
+          <h1 id="maintitle" class="text-orange">NIMPLAY jam 2023</h1>
+        </div>
+        <div class="home-p-box">
+          <p class="text-gray">
+            Indie video game company. dedicated to the development of apps and
+            games. Creating a community of developers for the creation of games,
+            jams and challenges to promote the development of Video Game
+            Industry.
+          </p>
+        </div>
+        <div class="home-button-text">
+          <p class="home-date text-gray">
+            Please contact us by Email for questions and suggestions.
+          </p>
+          <h3 class="email text-gray">nimrod7day@gmail.com</h3>
+        </div>
+      </section>
+    </div>
+    <section class="program">
+      <h2 id="program-title">
+        Nimplay<br />
+        2017
+      </h2>
+      <div class="red-line"></div>
+      <p class="home-date text-gray">
+        The logo of NIMPLAY video game development.
+      </p>
+      <div id="program-card">
+        <img src="./images/nimplayLogo.jpg" alt="" />
```

Comment:
Please add an alternative text to all of your images.

<img width="835" alt="Screen Shot 2022-06-23 at 9 05 25" src="https://user-images.githubusercontent.com/82426682/175318821-9d77ae30-37eb-4158-992c-c01e5ed51485.png">